server/api/profiles.js

```diff
@@ -46,9 +46,25 @@ router.get("/:profileId", (req, res) => {
 		})
 		.catch((err) => {
 			console.error(err);
-			res.send(500);
+			res.sendStatus(500);
+		});
+});
+//checks if email is not unique
+router.get("/:email", (req, res) => {
```

Comment:
there's a conflict in API url: there is already a GET with a parameter (router.get("/:profileId", (req, res)), you should rename the api router.get("/:email", (req, res) with router.get("/email/:email", (req, res)